src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -840,6 +841,18 @@ void Transformations::Lpt(const std::vector<ov::element::Type>& defaultPrecision
         },
         FuseConvertTransformation);
 
+    // Enable MatMulTransformation against FC nodes only
```

Comment:
I would say we need broader description why Int8 Matmul is not enabled for ARM at the moment. Plus ideally we should create feature request in oneDNN Github and add number here,

Reply:
Gave some more details here and added oneDNN ticket #